gen/fifo_gen.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// one_hot_test.dart
+// Test of one_hot codec.
+//
+// 2023 May 09
+// Author: Desmond Kirkpatrick <desmond.a.kirkpatrick@intel.com>
+//
+
+import 'dart:io';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/src/fifo.dart';
+
+Future<void> fifoGen() async {
+  final clk = SimpleClockGenerator(10).clk;
+  final reset = Logic()..put(0);
```

Comment:
You shouldn't need to do these `put`s since those are non-synthesizable deposits.